articles/user-guide/index.md

```diff
@@ -48,7 +48,7 @@ In the different topics of this guide, we detail the core concepts of the Q# lan
 
 ### Quantum Simulators and Resource Estimators
 
-- [Quantum simulators and host applications](xref:microsoft.quantum.machines): An overview of the different simulators available, as well as the general run model between host programs and target machines.
+- [Quantum simulators and host applications](xref:microsoft.quantum.machines): An overview of the different simulators available, as well as the general execution model between host programs and target machines.
```

Comment:
Style guide states that we shouldn't use the work "execution", hence they were changed to "run" before.

Reply:
@Bradben I see run wouldn't work. What would be the recommendation?

Reply:
This change was from https://github.com/MicrosoftDocs/quantum-docs-pr/pull/1064. This was in response to feedback from me that "run model" doesn't make sense to me. Alternatively something like "interaction model" could work.

By the way, I believe the style guide you are referring to is for the word "execute"? https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/execute. "Execution" is still allowed (I think for this very reason -- it may be a bit unclear when "run" is used as a noun.)

Reply:
Opened #1069 for this, please take a look there.